### ISSUE TITLE
Changing target directory

### ISSUE THREAD

**Issue by Cdik** [JeroenMols/LandscapeVideoCamera#54]
Hi,

First, thanks for this library :) It has been very useful for me.

Is there a way I can choose the target folder of the recorded video?
I have seen that your `VideoFile.class` is creating a _Movies_ directory with the `getFile()` function but how can I override it?

Thanks in advance.
Cedric

**Comment by JeroenMols** [comment]
Currently there is no way to do that, but that would seem like a meaningful feature for this library. So if you're up for building it, I'll gladly take a pull request.

**Comment by C-driiic** [comment]
Hi,

I have made a pull request _(my first pull request ever!)_ regarding the _"choose target directory"_ feature.

I tried to follow as much tutorials as I could to make this pull request as clear as possible but if I made a mistake or forgot something, my apologizes. 

(I made this pull request in your _master_ branch.... is that okay or was I supposed to use the _develop_ branch?)

First I wanted to implement like a _"folder chooser"_ but it was a bit too much of changes for a first pull request :/ so I have made sort of a _"create folder"_ feature which creates the folder if it does not exist, otherwise, does nothing than target the desired directory.

Here's a brief summary of what I have done:
- Added an optional `extra` to the `VideoCaptureActivity` as `EXTRA_OUTPUT_DIRECTORY`
    - this extra (a string) is the name of your folder _(or sub-folder, see below)_
- Added this feature to the example `CaptureDemoFragment`
    - as an `EditText` where you enter the name of your folder (or sub-folder if you enter e.g. _folder/sub-folder_). When the `EditText` doesn't have focus anymore, it's automatically filled with the entire external directory path (in grey the default root path followed by your folder/sub-folder name in black). When the `EditText` gain focus again, the default root path is cleared out, leaving only what you have typed in - to avoid confusions.
- I have also updated the Readme file.
- I haven't touched the version number/code. _I have absolutely no clue if I should have done it or not._
- I have updated the buildToolsVersion from 26.x to 27.x and the gradle version from 3.x to 4.x - _but again, I did that to suits my needs but don't really know if I was suppose to do that or not, or if it affects you somehow._

Voilà. Hope this is not too far from the good methodology :)
BR
Cedric
